### DISCUSSION TITLE
Sidebar

### DISCUSSION THREAD

**Discussion by bart-krakowski** [community/community#38421] [Other Feature Feedback, Questions, & Ideas]
The sidebar isn't scrollable, so I can't see its full content.

Details:
- Device: Macbook Pro 16" (1536 x 960px)

**Comment by AM2i9** [comment]
I'm having a similar if not the same issue. After clicking Show More under the Recent Repositories section of the sidebar, trying to scroll only scrolls the rest of the page. Only once I reach the bottom of my feed does it start scrolling the sidebar, and the sidebar extends below the footer.

**Comment by ossanna16** [comment]
Hi there @bart-krakowski and welcome to our community! Thank you for reporting this!
